autopush/router/apns2.py

```diff
@@ -21,6 +22,16 @@
 APNS_PRIORITY_LOW = '5'
 
 
+class ComplexEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, Decimal):
+            return int(float(obj.to_integral()))
```

Comment:
I'll blame pycharm a bit for this one. When I just used object, i was getting warnings. To be safe, i figured being explicit would be safer.